.github/actions/utility-action/action.yml

```diff
@@ -0,0 +1,112 @@
+name: Utility Actions
```

Comment:
General question: I feel like the functionality overlaps a lot with what we have in Rosa as well.
Do you see an issue with potentially making an opinioated "global" action out of this that is independent of the use case as it just installs and configures the same stuff that we need.

That way we would only have to maintain the logic ones.

Reply:
I don't see the issue managing this outside but sure let's move it to the point when we're aligning everything in one repo 👍